model/buildlogger_test.go

```diff
@@ -0,0 +1,200 @@
+package model
+
+import (
+	"testing"
+	"time"
+
+	"github.com/evergreen-ci/cedar"
+	"github.com/evergreen-ci/pail"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
```

Comment:
can these data be in scope in the test or produced by some kind of constructing function?